app/models/job.rb

```diff
@@ -1,4 +1,14 @@
 class Job < ActiveRecord::Base
   validates :title, :level_of_interest, presence: true
+  has_many :comments
   belongs_to :company
+  belongs_to :category
+
+  def ordered_comments
```

Comment:
the ordering of the actual comments should probably be a live in then `Comment` model because it has to do with the logic and the data that lives in a `Comment`